### DISCUSSION TITLE
How to send Token with HTTP Request

### DISCUSSION THREAD

**Discussion by dominikstohl** [lowdefy/lowdefy#1041] [Questions and Help] [answered]
Hello everyone,
is it possible to send the token of a user from OpenID i.e. as an Authorization Header via HTTP?

In the docs nothing is written about this and I looked at the lowdefy.user object in the console, but the token is not available.

I need to access a REST API and want to authenticate the user via the token.

Thanks for your help! Dominik

**Comment by SamTolmay** [comment, marked as answer]
Hi @dominikstohl 

Unfortunatly this isn't something that can be done yet. You can track this issue for this feature: #690 

We are making changes to how authentication works in version 4, so this feature will have to wait for those changes.

**Reply by dominikstohl** [reply]
Hey @SamTolmay, thanks for your answer! I'm excited about v4, thank you for this cool project!
